### ISSUE TITLE
Maya Tool - Solver UI - Add Right-Click Menu to Attributes

### ISSUE THREAD

**Issue by david-cattermole** [david-cattermole/mayaMatchMoveSolver#86]
We often need to perform operations on Attributes in the Maya Channel Box.

Common Operations:
- Lock / Unlock Attributes
- Set Keyframe
- Remove Keyframe
- Remove All Keyframes / Break Connection.
- Set Minimum Value *
- Set Maximum Value *

These operations should be available from the Solver UI, as right-click menu on the selected Attributes.

Usage:
1) Create Camera
2) Create Marker under Camera
3) Add Bundle attributes to the Solver UI.
4) Select Bundle Attributes in Solver UI.
5) Right-Click on Attributes in Solver UI, choose "Lock".
The selected attributes will be locked.

The "Set Minimum / Maximum Value" are for use with #65.

**Comment by david-cattermole** [comment]
Regarding the implementation in Qt, we can override the QWidget's contextMenuEvent virtual method to create a menu; 
https://doc.qt.io/qt-5/qwidget.html#contextMenuEvent

Here is a C++ example from the docs (https://doc.qt.io/qt-5/qtwidgets-mainwindows-menus-example.html):
```
void MainWindow::contextMenuEvent(QContextMenuEvent *event)
{
    QMenu menu(this);
    menu.addAction(cutAct);
    menu.addAction(copyAct);
    menu.addAction(pasteAct);
    menu.exec(event->globalPos());
}
```

**Comment by david-cattermole** [comment]
Right-click menus have been added to both the Objects and Attributes browsers in the Solver UI.

Two right-click options have been added for the Objects; 
* Swap Marker / Bundle Selection
* Select Both Markers and Bundles

For Attributes, these options have been added:
* Lock Attributes
* Unlock Attributes
* Set Keyframe
* Delete Keyframe
* Delete Keyframes (Timeline)
* Delete Static Channels
* Break Connections
* Bake Attributes
* Reset Values

In the Attributes browser, if a node is selected and the right-click menu is used, all attributes "under" the node in the UI will be affected. Note this includes attributes that are locked and hidden from the UI, but are still technically added to the UI.

This feature is expected to be released in the next version of mmSolver; v0.3.4.
